build/webpack.config.js

```diff
@@ -48,7 +48,8 @@ const config = Object.assign(commonConfig(), {
         contentBase: './test',
         disableHostCheck: true,
         host: '0.0.0.0',
-        inline: true
+        inline: true,
+        port: 8000
```

Comment:
I think this port will conflict with the one defined for box-content-preview's webpack server.

Reply:
Why can't they both run on 8000? They shouldn't be running at the same time.

Reply:
Possibly; I'm not sure. However, the scripts are hard-coded to look for 8000, so even if the port gets bumped to 8001, they will fail.